javascript_nodejs/5.multiturn-prompts-bot/dialogs/mainDialog/index.js

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { ActionTypes, MessageFactory } = require('botbuilder');
+
+const { TextPrompt, NumberPrompt, DialogSet } = require('botbuilder-dialogs');
+
+const USER_NAME_PROP = 'user_name';
+const AGE_PROP = 'user_age';
+const WHO_ARE_YOU = 'who_are_you';
+const HELLO_USER = 'hello_user';
```

Comment:
I've been using pascal cased values for constants, we should decide together what we want the samples to look like.

Ping @stevenic, @sgellock, @vishwacsena